recipe/meta.yaml

```
@@ -9,28 +9,34 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 0fe8a415cea8f340e7a9af9c54fc71a649b43e8ca3cc732986116b3cb135d303
+  sha256: b3ef57c62535b0941697cce638c08900d87fcb67e29cfa99e8a68f747f393f7a
+  patches:
+    - 0001-do-not-depend-on-hatch-fancy-pypi-readme.patch
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-build-isolation -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   skip: true  # [py<37]
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - python
     - cython 0.29
     - pip
     - setuptools
     - wheel
+    - hatchling
   run:
     - python
-    - typing-extensions >=4.2.0
+    - typing-extensions >=4.6.1
+    - annotated-types >=0.4.0
+    - pydantic-core ==2.14.6
   run_constrained:
     - email-validator >=1.0.3
```

```suggestion
    - email-validator >=2.0.0
```
https://github.com/pydantic/pydantic/blob/v2.5.3/pyproject.toml#L75